CHANGELOG.md

```diff
@@ -366,6 +402,19 @@ Branch: `nilrt/22.5/hardknott`
 - [Removed](https://github.com/ni/meta-nilrt/pull/290) packages dropped from upstream.
 
 
+----
+## 8.20
```

Comment:
This is 8.22 release. Please add "No changes." for 8.20 and 8.21 - refer 8.19 for how to do this.

Reply:
Fixed.